packages/core/stories/card/card.doc.stories.mdx

```diff
@@ -26,12 +26,26 @@ This default example is intended to show the way the Card component’s content
   <Story id="core-card--default" />
 </Canvas>
 
+## Adding actions
+
+If you intend to add any links or buttons within your Card, only add them into a default Card, not an interactable Card, to ensure ADA compliancy.
```

Comment:
Should we add advice/a story on how to add link styling to a "link" in interactable cards, like on the current site? @joshwooding @ivan-calderon

Reply:
It depends if it's a valid pattern. I'll let @ivan-calderon answer